doc/how-to/vshard_quick.rst

```diff
@@ -1,160 +1,525 @@
+    *   :ref:`vshard.router.callrw() <router_api-callrw>`: Inserts a tuple to a storage identified the generated bucket ID.
+
+3.  Create the ``get`` function for getting data:
+
+    ..  literalinclude:: /code_snippets/snippets/sharding/instances.enabled/sharded_cluster/router.lua
+        :language: lua
+        :start-at: function get
+        :end-before: function insert_data
+        :dedent:
+
+    Inside this function, :ref:`vshard.router.callro() <router_api-callro>` is called to get data from a storage identified the generated bucket ID.
+
+4.  Finally, create the ``insert_data()`` function that inserts sample data into the created space:
+
+    ..  literalinclude:: /code_snippets/snippets/sharding/instances.enabled/sharded_cluster/router.lua
+        :language: lua
+        :start-at: function insert_data
+        :dedent:
+
+The resulting ``router.lua`` file should look as follows:
+
+..  literalinclude:: /code_snippets/snippets/sharding/instances.enabled/sharded_cluster/router.lua
+    :language: lua
+    :dedent:
+
+
+
+..  _vshard-quick-start-build-settings:
+
+Configuring build settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Open the ``sharded_cluster-scm-1.rockspec`` file and add the following content:
+
+..  literalinclude:: /code_snippets/snippets/sharding/instances.enabled/sharded_cluster/sharded_cluster-scm-1.rockspec
+    :language: none
+    :dedent:
+
+The ``dependencies`` section includes the specified version of the ``vshard`` module.
+To install dependencies, you need to :ref:`build the application <vshard-quick-start-building-app>`.
+
+
+.. _vshard-quick-start-building-app:
+
+Building the application
+------------------------
+
+In the terminal, open a directory where the :ref:`tt environment is created <vshard-quick-start-creating-app>`.
+Then, execute the ``tt build`` command:
+
+.. code-block:: console
+
+    $ tt build sharded_cluster
+       • Running rocks make
+    No existing manifest. Attempting to rebuild...
+       • Application was successfully built
+
+This installs the ``vshard`` dependency defined in the :ref:`*.rockspec <vshard-quick-start-build-settings>` file to the ``.rocks`` directory.
+
+
+
+..  _vshard-quick-start-working-cluster:
+
+Working with the cluster
+------------------------
+
+.. _vshard-quick-start-working-starting-instances:
+
+Starting instances
+~~~~~~~~~~~~~~~~~~
+
+To start all instances in the cluster, execute the ``tt start`` command:
+
+.. code-block:: console
+
+    $ tt start sharded_cluster
+       • Starting an instance [sharded_cluster:storage-a-001]...
+       • Starting an instance [sharded_cluster:storage-a-002]...
+       • Starting an instance [sharded_cluster:storage-b-001]...
+       • Starting an instance [sharded_cluster:storage-b-002]...
+       • Starting an instance [sharded_cluster:router-a-001]...
+
+
+.. _vshard-quick-start-working-bootstrap:
+
+Bootstrapping a cluster
+~~~~~~~~~~~~~~~~~~~~~~~
+
+To bootstrap the cluster, follow the steps below:
```

Comment:
Maybe add an intro sentence that bootstrap is required? It's not obvious.

Reply:
Not sure - all the steps in this tutorial are required :) I've created a separate section for such a simple operation to emphasise this.

Reply:
Rephrased:

```
After starting instances, you need to bootstrap the cluster as follows:
```